spec/forms/idv/api_image_upload_form_spec.rb

```diff
@@ -28,11 +28,28 @@
   let(:back_image) { DocAuthImageFixtures.document_back_image_multipart }
   let(:selfie_image) { nil }
   let(:liveness_checking_required) { false }
+  # rubocop:disable Lint/InterpolationCheck
+  let(:front_image_file_name) { '\n"#{ENV}".jpg' }
```

Comment:
Part of what this linter is checking for is if the interpolation is valid. `#{...}` has to be used within double-quotes, not single quotes, so this is literally the string `'\n"#{ENV}".jpg'`. Is that intended?

What are we trying to test with the special characters in these names?

Reply:
@aduth , use normal string now.